content/posts/customizing/add-section.md

```diff
@@ -10,4 +10,76 @@ menu:
     weight: 410
 ---
 
-## Coming soon.......
\ No newline at end of file
+While the [configuration section](/posts/configuration) goes in detail on configuring each pre-defined section, this post will outline in detail how to add your own fully custom section and include it on the home page.
+
+### Background
+
+The existing "sections" for this theme are configured by the theme consumer via a `.yml` file in the `./data/<locale>/sections` folders, and that data is then used by the theme's internal templates to provide the each of the stacked horizontal "sections" on the home page:
+
+* About
+* Skills
+* Experiences
+* Projects
+* Recent Posts
+* Achievements
+
+
+Each of these existing sections have a set of common data used for displaying the sections in the correct order, providing headers, linking to them, etc. They also then have data structures specific to the template used to render the actual sections contents.
+
+### Configuring a Custom Section
+
+Configuring a custom section is as simple as providing a new `.yml` file for your custom section in your `./data/<locale>/sections` folder, and writing your own custom template for it. Lets step through an example piece by piece assuming I want to make a new section called `custom`.
+
+#### Required Configs
+
+Firstly, you must create a new `.yml` configuration file in your `data/<locale>/sections/` folder. In this example, we'll call it custom.yml
+
+In this file you MUST provide the following config fields (the values can be what you want though)
+
+```yml
+section:
+  name: Your Custom name used for the section header by default
+  id: your-template-name
```

Comment:
The internals seem to use the ID and optionally provide a template override. i thought i'd keep it simple and not expose variations and fallbacks, but I can go into more detail here if you'd like. Or if anyone opens an issue we can update to document other cases

Reply:
I agree. However, for some cases template override information will be helpful. For example, if we add two gallery templates, user can use id as `gallery` and provide the template override.